### PR TITLE
Setup bors

### PR DESCRIPTION
**Pull request by tcharding**
Add bors, same as we do in comit-rs.

Also adds the other two badges we have in comit-rs README: CircleCI, and unsafe forbidden.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,9 @@
 # Blockchain contracts
+
+[![CircleCI](https://circleci.com/gh/comit-network/comit-rs.svg?style=svg)](https://circleci.com/gh/comit-network/blockchain-contracts)
```
Comment (D4nte) [inline README.md]:
```suggestion
[![CircleCI](https://circleci.com/gh/comit-network/blockchain-contracts.svg?style=svg)](https://circleci.com/gh/comit-network/blockchain-contracts)
```

Comment (tcharding) [reply README.md]: Thanks, fixed.

**Comment by thomaseizinger** [comment]
Hold on, do we only want the beta build to succeed or the stable on aswell?

**Comment by D4nte** [comment]
> Hold on, do we only want the beta build to succeed or the stable on aswell?

We want the build that matches comit-rs toolchain.

**Comment by tcharding** [comment]
> Please double check if we actually deny unsafe :)

We have in `src/lib.rs`

`#![forbid(unsafe_code)]`

That's all we need right?

**Comment by tcharding** [comment]
>> Hold on, do we only want the beta build to succeed or the stable on aswell?

> We want the build that matches comit-rs toolchain.

Right, now I'm out of my depth.  The circleci config files from blockchain-contracts and comit-rs are sufficiently different, and contain too many fields that I do not fully understand, that I cannot complete this task without help.  `beta-build` was my best guess?  Everyone in the office is madly working on the workshop and this has low priority so leaving like this for now.

**Comment by D4nte** [comment]
`beta-test` is what you put down and it is what we want.
comit-rs is using beta channel at the moment so we want to ensure that blockchain_contracts builds on beta.

Once we revert to stable on comit-rs, we will need to revert to stable here.

Please note GitHub marks the PR green if beta-build and beta-test are green.

So all good, let's merge!

File: README.md (hunk 1 above)
Comment (Nguyentiensy99) [reply README.md]: Có thể thêm vào realme của tôi hợp nhất vào di động và thêm tên tôi ok
???

**Pull request merged**